File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/NewTeamTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class NewTeamTag extends Tag {
    public NewTeamTag(int count) {
        super(count, Constants.NEW_TEAM_TAG, Constants.NEW_TEAM_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/NewTeamTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class NewTeamTag(count: Int) : Tag(count, Constants.NEW_TEAM_TAG, Constants.NEW_TEAM_TAG_REMOTE, HashSet(listOf(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))